build/config.props

```diff
@@ -36,8 +36,12 @@
     <VsTargetChannel Condition="'$(IsEscrowMode)' == 'true'">int.d$(VsTargetMajorVersion).$(MinorNuGetVersion)</VsTargetChannel>
     <!-- Specifies the SDK version to download to use for testing. The first value represents the channel, the second value represents the exact SDK version to be download. If a version is not specified, the latest version from the channel will be downloaded.
     Note that multiple SDKs can be downloaded by using `;` as a separator.
+    The channel needs to be two-part version in A.B format. The version needs to be the exact version number if specified.
+    e.g. 5.0;6.0 means install the latest versions from both channel 5.0 and channel 6.0.
+         6.0:6.0.100-preview.7.21379.14 means install the preview version 6.0.100-preview.7.21379.14.
+    Refer to https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script#options for more details.
     -->
-    <CliBranchForTesting Condition="'$(CliBranchForTesting)' == ''">release/5.0.2xx:5.0.200-servicing.21120.4</CliBranchForTesting>
+    <CliBranchForTesting Condition="'$(CliBranchForTesting)' == ''">6.0.3xx</CliBranchForTesting>
```

Comment:
Should be put a comment next to the major/minor version numbers near the top, since every major/minor version maps to a different .NET SDK version?

Reply:
Thanks! How about adding "Ideally, this is the same .NET SDK version NuGet inserts into." before CliBranchForTesting?